### PR TITLE
feat: init

### PR DESCRIPTION
**Pull request by michael-lynch**
## Description

A new React app has been created using Create React App (CRA) and the following packages have been added:
- `semantic-ui-css`
- `semantic-ui-react`

The existing `<Theme>` page has been added but instead of using a Semantic UI theme, individual CSS files have been added to a `/styles` directory that are imported into the `src/ga.css` file, which is the file that can be shared with FAST.

_Note that the individual CSS files are currently blank and will need to be populated in subsequent PRs. These files should generally mirror the overrides files we see in Semantic themes._

## Motivation

FAST doesn't seem to be using Semantic themes, so it's likely more straight forward if we just use our own CSS files, rather than using the Semantic theming system.

## Screenshots

Screenshot 2023-05-17 at 5 28 41 PM

## Testing

```
npm install
npm start
```

Have a look at the folder structure, the individual CSS files and the `ga.css` file that they are imported into.

### CODE REVIEW

File: src/ga.css
Comment (michael-lynch) [inline src/ga.css:1]: This is the main CSS file that can be shared with FAST. We should wait until it is merged into `develop` and then we can send the URL.

Note that this file is comprised of smaller, individual files that are imported into it. These files should generally mirror the Semantic theme overrides files. It is in these files that we can target `.ui` classes and override the styles we need to.

**Review by Dart-EY — APPROVED** [review]

**Pull request merged**
